polymesh_schema.json

```diff
@@ -626,7 +626,14 @@
             "leg_id": "u64",
             "signer": "AccountId",
             "signature": "OffChainSignature"
-        }
+        },
+        "WeightToFeeCoefficient": {
+            "coeffInteger": "Balance",
+            "coeffFrac": "Perbill",
+            "negative": "bool",
+            "degree": "u8"
+        },
```

Comment:
Are we facing issues with this type at Polkadot explorer? It should be a primitive type of substrate so we don't need to define it.

If I am not wrong we also don't need the StakingLedger type as we upgrade the staking pallet with the recent code. 

am I missing something here?

Reply:
You are right, we don't need `StakingLedger` anymore. `WeightToFeeCoefficient` is required to make it work with the older version of the app that we are hosting. I'll remove `StakingLedger`